packages/web-config-server/src/apiV1/measureData.js

```diff
@@ -263,8 +267,10 @@ export default class extends DataAggregatingRouteHandler {
       ...restOfPresentationConfig
     } = config;
 
-    const { dataSourceType = DATA_SOURCE_TYPES.CUSTOM, periodGranularity } =
-      measureBuilderConfig || {};
+    const {
+      dataSourceType = legacy ? DATA_SOURCE_TYPES.SINGLE : DATA_SOURCE_TYPES.CUSTOM,
```

Comment:
You know I just looked into this some more, and I can't see it being used anywhere except the bit that fetches options from DHIS2 below. Let's just pop that into a `legacy === true` along with this destructuring, and then we can avoid introducing the semi-confusing `CUSTOM` type.